### ISSUE TITLE
SHELLCODE CONTAINS NULL BYTES DUE TO 127.0.0.1

### ISSUE THREAD

**Issue by jiachen247** [Re4son/slae-assignment2#1]
SHELLCODE CONTAINS NULL BYTES DUE TO 127.0.0.1

**Comment by Re4son** [comment]
Correct, and that code is not supposed to be used without an encoder. It's part of this blog post:

https://whitedome.com.au/re4son/slae-2-reverse-shell-72bytes/
